lib/hanami/action/rack.rb

```diff
@@ -213,16 +205,6 @@ def response
         [ @_status || DEFAULT_RESPONSE_CODE, headers, @_body || DEFAULT_RESPONSE_BODY.dup ]
       end
 
-      # Calculates an unique ID for the current request
-      #
-      # @return [String] The unique ID
-      #
-      # @since 0.3.0
-      def request_id
```

Comment:
⚠️ This cannot be removed, it's public API and IIRC used by `hanami`.

Reply:
No problem I will restore it 😄